arch/arm/src/stm32f7/stm32_can.c

```diff
@@ -658,15 +665,16 @@ static int stm32can_setup(FAR struct can_dev_s *dev)
   FAR struct stm32_can_s *priv = dev->cd_priv;
   int ret;
 
-  caninfo("CAN%d RX0 irq: %d RX1 irq: %d TX irq: %d\n",
+  caninfo("CAN%" PRId8 " RX0 irq: %" PRId8 " RX1 irq: %" PRId8
+          " TX irq: %" PRId8 "\n",
```

Comment:
```suggestion
  caninfo("CAN%" PRIu8 " RX0 irq: %" PRIu8 " RX1 irq: %" PRIu8
          " TX irq: %" PRIu8 "\n",
```
`port`, `canrx` and `cantx` are of type `uint8_t`
```
  uint8_t  port;     /* CAN port number (1 or 2) */
  uint8_t  canrx[2]; /* CAN RX FIFO 0/1 IRQ number */
  uint8_t  cantx;    /* CAN TX IRQ number */
```

Reply:
@gustavonihei @antmerlino - Thnak you both!

I will fix these is the AM, squash and force push.